CHANGELOG.rst

```diff
@@ -20,6 +20,7 @@ Features
 - CMake:
 
   - add ``openPMD::openPMD`` alias for full-source inclusion #277
+  - include internally shipped pybind11 v2.2.3 #281
```

Comment:
We should probably note somewehere that the (currently) included version includes modifications w.r.t. the actual ```pybind11 v2.2.3```. This would be a good place.

Reply:
the modifications I did are only relevant for direct inclusion (and to save some space: ~~tests~~).
if one switches to `openPMD_USE_INTERNAL_PYBIND11=OFF` with the exact same release, they are not needed.

Therefore, I think the note in the commit message is good enough if one wants to see what was added when.